templates/zerver/api/changelog.md

```diff
@@ -27,6 +27,9 @@ format used by the Zulip server that they are interacting with.
 /realm/user_settings_defaults`](/api/update-realm-user-settings-defaults): Added
   user `email_address_visibility` setting which will replace the existing realm
   `email_address_visibility` setting.
+* [`GET /events`](/api/get-events): Added user `email_address_visibility` field
+  to the person object in the events sent when adding a user and when user changes
+  their avatar.
```

Comment:
Is this only in these events, or also in the `/register` response? Is it an internal implementation detail because we need it for `apply_events`, or do clients need it too?

Reply:
Yes, `email_address_field` is explicitly included in these events to decide whether we need to include `avatar_url` or client can compute themseleves.